src/components/Basics/BasicsLayout.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <v-sheet rounded="lg" class="text">
+    <h1>Git Basics</h1>
+    <p>
+      In this section, we're going to explain the basic commands to work with
+      Git in any project.
+    </p>
+    <h2>Git Config</h2>
+    <p>
+      The first thing you'll need to do when installing Git is set your username
+      and email address. This is important because Git commits use this
+      information, and it is immutable in the commits you send:
+    </p>
+    <v-badge left overlap bordered icon="mdi-console-line" color="#f05033">
```

Comment:
@JuanJosePG Is possible apply the color with css? If you apply the color with css, after will be easier apply the color with variables

Reply:
"v-badge" tag is under development, after this PR, I'll create a common component for that and I'll write it better.

Reply:
This is solved